src/Backend/Modules/Search/Layout/Templates/Settings.html.twig

```diff
@@ -73,7 +73,7 @@
               <th>{{ 'lbl.Module'|trans|ucfirst }}</th>
               <th>
                 {{ 'lbl.Weight'|trans|ucfirst }}
-                <abbr class="fa fa-question-circle" data-toggle="tooltip" title="{{ 'msg.HelpWeight'|trans|ucfirst }}"></abbr>
+                <abbr class="fa fa-question-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpWeight'|trans|ucfirst }}"></abbr>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320